plugins/inputs/minecraft/minecraft.go

```diff
@@ -0,0 +1,137 @@
+package minecraft
+
+// minecraft.go
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const sampleConfig = `
+  # server address for minecraft
+  server = "localhost"
+  # port for RCON
+  port = "25575"
+  # password RCON for mincraft server
+  password = "replace_me"
```

Comment:
This is done inconsistently in our current plugins, but comments should have two hashes, items with default values should have a single hash, and items that must be filled should be with no hashes.